mono/mini/method-to-ir.c

```diff
@@ -12754,11 +12754,11 @@ mono_spill_global_vars (MonoCompile *cfg, gboolean *need_local_opts)
 	memset (spec2, 0, sizeof (spec2));
 
 	/* FIXME: Move this function to mini.c */
-	stacktypes ['i'] = STACK_PTR;
-	stacktypes ['l'] = STACK_I8;
-	stacktypes ['f'] = STACK_R8;
+	stacktypes [(int)'i'] = STACK_PTR;
```

Comment:
Warnings only, but this preserves meaning.
'a'  is a char in C++, int in C.
Without the cast, the type of the array index has been changed.
Casting restores the old meaning.